src/Editor/GUI/SelectHymnWindow.cpp

```diff
@@ -26,12 +26,12 @@ void SelectHymnWindow::Show() {
         
         ImGui::InputText("Name", name, 128);
         
-        if (ImGui::Button(openButtonName, ImVec2(120,0))) {
+        if (ImGui::Button(openButtonName, ImVec2(120, 0))) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)